README.md

```diff
@@ -213,7 +213,48 @@ executable `taskDPL`.
 
 ## Data Sampling
 
-TODO -> piotr
+Data Sampling provides possibility to sample data in DPL workflows, basing on certain conditions ( 5% randomly, when payload is greater than 4234 bytes, etc.). The job of passing the right data is done by a data processor called `Dispatcher`. A desired data stream is specified in form of Data Sampling Policies, configured by JSON structures. Please refer to the main [Data Sampling readme](https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/README.md#data-sampling) for more detailed information.
```

Comment:
"The Data Sampling provides the possibility to sample data in DPL workflows, based on certain conditions"